src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -19,12 +19,16 @@ public static class TextLoaderSaverCatalog
         /// <param name="hasHeader">Whether the file has a header.</param>
         /// <param name="separatorChar">The character used as separator between data points in a row. By default the tab character is used as separator.</param>
         /// <param name="dataSample">The optional location of a data sample. The sample can be used to infer column names and number of slots in each column.</param>
+        /// <param name="allowSparse">Whether the file can contain numerical vectors in sparse format.</param>
+        /// <param name="allowQuoting">Whether the file can contain column defined by a quoted string.</param>
```

Comment:
`ReadFromTextFile` calls these options:

```
            bool allowQuotedStrings = TextLoader.Defaults.AllowQuoting,
            bool supportSparse = TextLoader.Defaults.AllowSparse,
```

We should be consistent in the names everywhere.

Reply:
Ok. I also checked other `quote` and `sparse` in this file.

---
In reply to: [259422639](https://github.com/dotnet/machinelearning/pull/2630#discussion_r259422639) [](ancestors = 259422639)